src/SUMMARY.md

```diff
@@ -3,14 +3,14 @@
 - [About the documentation](./about_docs.md)
 - [Tutorials](./tutorials.md)
   - [Getting Started](./getting_started.md)
-  - [The many forms of "Hello, World!"](./many_forms_hello_world.md)
-  - [More advanced examples](./advanced_examples.md)
+  - [Examples](./examples.md)
 - [How-to Guides](./how_to.md)
   - [Transpile to Javascript](./transpile_js.md)
   - [Contribute to Alan](./contributing.md)
 - [Topic Guides](./topics.md)
   - [About Alan](./about_alan.md)
   - [Syntax and File Structure](./file_structure.md)
+    - [The many forms of "Hello, World!"](./many_forms_hello_world.md)
```

Comment:
I'm not entirely sold on moving this here. What's your thinking behind this?

Reply:
Would you rather fully remove it? I'm also down to do that. Don't have a strong opinion. To me it provides interesting background information about syntax and file structure. From the first sentence

> The classic "Hello, World!" examples are a simple way to understand the minimum of structure you need to have in any code you write in a given language, with Java being famously heavyweight and Python famously featherweight.

Reply:
Okay, but maybe we trim a few of the variants out at the same time for this new focus?